makefile

```diff
@@ -0,0 +1,53 @@
+GO_VERSION ?= 1.20.2
```

Comment:
This file `makefile` can be removed as the `Makefile` has already been merged into the main branch

Reply:
resolved